Decred Wallet/Features/Custom Dialogs/ImportWatchWalletInput.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  ImportWatchWalletInput.swift
+//  Decred Wallet
+//
+// Copyright (c) 2018-2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import Foundation
+import UIKit
+
+typealias ImportWatchWalletInputDialogCallBack = (_ publicKey: String, _ dialogDelegate: ImportWatchWalletInputDelegate?) -> Void
+
+protocol ImportWatchWalletInputDelegate {
+    func dismissDialog()
+    func displayError(errorMessage: String)
+}
+
+class ImportWatchWalletInput: UIViewController {
+    @IBOutlet weak var inputKeyTextField: FloatingPlaceholderTextField!
+    @IBOutlet weak var importButton: Button!
+    @IBOutlet weak var warningLabel: UILabel!
+    
+    private var callback: ImportWatchWalletInputDialogCallBack!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.inputKeyTextField.placeholder = "Extended public key"
```

Comment:
why not use SimpleTextInputDialog? instead of creating a new dialog that has the same features, just add the features that is missing.